<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunDialog</class>
 <widget class="QDialog" name="RunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RETK - Run</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources.qrc">
    <normaloff>:/logo</normaloff>:/logo</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="optionList">
       <item>
        <property name="text">
         <string>Create Dumb Pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Request Updates</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btOk">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btQuit">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
